templates/lustrum12/thema.html.twig

```diff
@@ -0,0 +1,164 @@
+			var scrollMaxY = window.scrollMaxY || (document.documentElement.scrollHeight - document.documentElement.clientHeight)
+			var vertical_position = pageYOffset || document.documentElement.scrollTop;
+			var percentile = (vertical_position / scrollMaxY)
+
+			var zandloper = document.getElementById("zandloper");
+			var heel = document.getElementById("zandloper_heel");
+			var kapot = document.getElementById("zandloper_kapot");
+
+			zandloper.style.top = percentile * 100 + '%';
+			zandloper.style.marginTop = -50 - (20 * percentile) + "px";
+
+			if (percentile < 0.99) {
+				heel.style.display = "";
+				kapot.style.display = "none";
+			} else {
+				heel.style.display = "none";
+				kapot.style.display = "";
+			}
+		};
+
+		window.addEventListener('load', function () {
+			document.getElementById("game-container").innerHTML =
+				`<iframe class="unity-container" src="https://jobkanis.github.io/lustrumgame/" allow="fullscreen" scrolling="no"></iframe>`
+		});
+
+		window.addEventListener('load', function () {
+			document.getElementById("player").innerHTML =
+				`<iframe src="https://www.youtube-nocookie.com/embed/9rXmPGNczdE?controls=0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>`
+		});
+
+	</script>
+</head>
+
+<body class="zijbalk_verborgen">
+
+	<div id="zandloper" class="zandloper">
+		<img id="zandloper_kapot" class="kapot" src="/images/lustrum12/ZandloperKapot.png"/>
+		<img id="zandloper_heel" class="heel" src="/images/lustrum12/ZandloperHeel.png"/>
+	</div>
+
+	<nav class="site-header">
+		<a class="logo-titel d-inline-block" href="./">
+			<img src="/images/lustrum12/themaheader.png"
+					 alt="Logo menu" />
+		</a>
+		<div class="container">
+			<div class="menu d-flex flex-column flex-md-row justify-content-between align-items-center">
+				<a class="d-none d-md-inline-block" href="./">Home</a>
+				<a class="d-none d-md-inline-block" href="#top">Top</a>
+				<a class="d-none d-md-inline-block" href="#themafilm">Themafilm</a>
+				<a class="d-none d-md-inline-block" href="#game">Game</a>
+				<a class="d-none d-md-inline-block" href="#instagram">Instagram</a>
+				<a class="d-none d-md-inline-block" href="#commissie">Commissie</a>
+			</div>
+		</div>
+	</nav>
+
+	<a id="top"></a>
+	<div class="thema thema-container">
+		<img class="thema-plaetje" src="/images/lustrum12/thematitel.png" alt="Lustrum Instagram" >
+	</div>
+
+	<div class="divider">
+		<img class="boven" src="/images/lustrum12/DividerBoven.png">
+	</div>
+
+	<div class="video">
+		<a class="anchor glow" id="themafilm"></a>
+		<!-- <div class="kopje">
+				<img class="plaetje" src="${require('../../assets/images/KopjeThemafilm.png')}"/>
+		</div> -->
+		<h2 class="titel">Themafilm</h2>
+		<div class="container">
+			<div class="video-container">
+				<div class="youtube-container closed">
+					<div class="embed-responsive embed-responsive-16by9" id="player"></div>
```

Comment:
class moet zijn: `ratio ratio-16x9`, wat je nu hebt is voor bootstrap 4

https://getbootstrap.com/docs/5.1/helpers/ratio/